G/GR/build_tarballs.jl

```diff
@@ -99,7 +99,7 @@ dependencies = [
     Dependency("Libtiff_jll"; compat="~4.5.1"),
     Dependency("Pixman_jll"),
     HostBuildDependency("Qt6Base_jll"),
-    Dependency("Qt6Base_jll"; compat="~6.4.2"), # Never allow upgrading more than the minor version without recompilation
+    Dependency("Qt6Base_jll"; compat="~6.5.2"), # Never allow upgrading more than the minor version without recompilation
```

Comment:
@barche is it OK? Do you know what was the problem yesterday?

Reply:
For sure it should be 6.5.2 so GR.jl and QML.jl can be used together, I'll take a look at the errors.

The problem yesterday was using GR_jll binaries compiled against Qt 6.4.2 together with Qt6Base_jll version 6.5.2, which normally should have been prevented by the compat entry, except that the compat entry was not strict enough.